internal/generate/olm-catalog/package_manifest.go

```diff
@@ -26,16 +26,15 @@ import (
 	"github.com/operator-framework/api/pkg/validation"
 	"github.com/operator-framework/operator-registry/pkg/registry"
 	"github.com/operator-framework/operator-sdk/internal/generate/gen"
+	"github.com/operator-framework/operator-sdk/internal/scaffold"
 	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
 
 	"github.com/ghodss/yaml"
 	log "github.com/sirupsen/logrus"
 )
 
 const (
-	PackageManifestFileExt = ".package.yaml"
-
-	ManifestsDirKey = "manifests"
+	packageManifestFileExt = ".package.yaml"
```

Comment:
I'll see but that might be difficult I think.
Since we run both CSV and package manifest generators as part of `generate csv` cmd I need to make sure both of them work correctly per the `--inputs` and `--output-dir` flag changes.